dao/src/main/java/uk/gov/moj/sdt/dao/IndividualRequestDao.java

```diff
@@ -169,18 +169,17 @@ public List<IIndividualRequest> getStaleIndividualRequests(final int minimumAgeI
         final LocalDateTime now = LocalDateTime.now();
         final LocalDateTime latestTime = now.minusMinutes(minimumAgeInMinutes);
         List<IndividualRequest> queryList = this.queryAsList(IndividualRequest.class, () -> {
-            Predicate[] sdtCustomerPredicate =   createIndividualRequestPredicate(false, latestTime);
+            Predicate[] sdtCustomerPredicate =   createIndividualRequestPredicate(latestTime);
```

Comment:
Replace the double space after the equals sign with a single space.

Reply:
done